README.md

```diff
@@ -610,6 +613,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 * [Dropbox](https://www.dropbox.com/) - Dropbox app offers easy screenshot capturing and sharing ![Freeware][Freeware Icon]
 * [Snagit](https://www.techsmith.com/screen-capture.html) - Screen Capture and Recording Software. Simple and Powerful.
 * [Screen Studio](https://www.screen.studio/) - Record beautiful screens in minutes, with built-in exquisite frame animations, no need for editing.
+* [Zappy](https://zapier.com/zappy) - Zappy is a screenshot and screen recording app all in one. Has some simple editing tools built in.
```

Comment:
@smit309 Repeated

- https://github.com/jaywcjlove/awesome-mac/pull/1365

Reply:
Sorry but I don't understand what you are saying. Can you describe more

Reply:
@smit309 `Zappy` has been merged into the manifest

Reply:
I did Add RapidAPi , Add Unsplash wallpapers and Add Capacities but it is not merged

Reply:
![Screenshot 2024-01-07 104108](https://github.com/jaywcjlove/awesome-mac/assets/103751285/79302c9d-12af-48a4-ab87-0dbf92d1f249)
![Screenshot 2024-01-07 104031](https://github.com/jaywcjlove/awesome-mac/assets/103751285/4ac530a8-d2bb-41d1-8c68-af4bc8784042)

Reply:
@smit309 Have been merged. 

- https://github.com/jaywcjlove/awesome-mac/pull/1365